Doc/library/os.rst

```diff
@@ -3710,8 +3710,8 @@ written in Python, such as a mail server's external command delivery program.
    The positional-only arguments *path*, *args*, and *env* are similar to
    :func:`execve`.
 
-   The *path* parameter is the path to the executable file.The *path* should
-   contain a directory.Use :func:`posix_spawnp` to pass an executable file
+   The *path* parameter is the path to the executable file. The *path* should
+   contain a directory. Use :func:`posix_spawnp` to pass an executable file
```

Comment:
Nice catch! A minor nitpick I have is that [PEP 12](https://www.python.org/dev/peps/pep-0012/#id27) states:

> You must adhere to the Emacs convention of adding two spaces at the end of every sentence

Although I realized most of the docs are inconsistent with this and it's not strictly adhered to, I thought I'd just point it out.

```suggestion
   The *path* parameter is the path to the executable file.  The *path* should
   contain a directory.  Use :func:`posix_spawnp` to pass an executable file
```

Reply:
I'm not sure why the two spaces convention is useful for, but it can be automated easily. I'm adding another commit for this below, feel free to `squash` them into one.

```bash
sed -i "s/\.\ \([A-Z]\)/\.\ \ \1/g" os.rst
```

Reply:
Yeah, I don't use Emacs so I'm not able to justify them either. Personally, I'd recommend the automated adding of spaces in a separate PR (and just add the spaces for your own edits for this one since it fixes a visual error). Changes that fix a styling problem should probably go into another PR https://devguide.python.org/pullrequest/#making-good-commits. Sorry if I created confusion regarding what was to be fixed!

Reply:
Dropped the second commit, added double spaces only to the changed lines.